packages/core/src/utils/pipeHandler.ts

```diff
@@ -5,6 +5,29 @@ import { PlateEditor } from '../types/PlateEditor';
 import { DOMHandlers, HandlerReturnType } from '../types/plugin/DOMHandlers';
 import { TEditableProps } from '../types/slate-react/TEditableProps';
 
+function convertDomEventToSyntheticEvent(domEvent: Event) {
+  const syntheticEvent = {
+    ...domEvent,
+    nativeEvent: domEvent,
+    currentTarget: domEvent.currentTarget as EventTarget,
+    target: domEvent.target as EventTarget,
+    bubbles: domEvent.bubbles,
+    cancelable: domEvent.cancelable,
+    defaultPrevented: domEvent.defaultPrevented,
+    eventPhase: domEvent.eventPhase,
+    isTrusted: domEvent.isTrusted,
+    timeStamp: domEvent.timeStamp,
+    type: domEvent.type,
+    isDefaultPrevented: () => domEvent.defaultPrevented,
+    isPropagationStopped: () => false,
```

Comment:
Can you implement `isPropagationStopped` using a local variable in `convertDomEventToSyntheticEvent`'s scope that gets set to true when `stopPropagation` is called?